src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -1059,11 +1070,15 @@ private int countRunningReapers() {
 
     @Override
     protected Set<String> initialize() {
-      Collection<RepairParameters> ongoingRepairs = context.storage.getOngoingRepairsInCluster(clusterName);
+      Collection<RepairParameters> ongoingRepairs =
+          context.storage.getOngoingRepairsInCluster(clusterName);
       Set<String> busyHosts = Sets.newHashSet();
-      ongoingRepairs.forEach((ongoingRepair) -> {
-        busyHosts.addAll(coordinator.tokenRangeToEndpoint(ongoingRepair.keyspaceName, ongoingRepair.tokenRange));
-      });
+      ongoingRepairs.forEach(
+          (ongoingRepair) -> {
+            busyHosts.addAll(
+                coordinator.tokenRangeToEndpoint(
+                    ongoingRepair.keyspaceName, ongoingRepair.tokenRange));
+          });
```

Comment:
trivial: 
 did anything actually changed in the `initialize()` method?
 can you take it out of the PR please. it's not a great precedence to encourage.